main.go

```diff
@@ -17,11 +17,13 @@ limitations under the License.
 package main
 
 import (
+	goflags "flag"
```

Comment:
nit: this is probably an indication that we either shouldn't use the name `flag` OR the flags should be exported from another package; e.g. `pkg/generators`?

Reply:
We have flags imported internally here: https://github.com/kcp-dev/code-generator/blob/4977dcfd75f4ea485070682f01db1fc851ceb666/main.go#L28. Did you mean to rename those instead of tagging this as `go flags`?